RealEstate/apps/core/templates/core/base.html

```diff
@@ -77,6 +77,12 @@
       </div><!-- /.container-fluid -->
     </nav>
     <div class="container">
+      {% block messages %}
+        <div class="row">
```

Comment:
Not a big deal, but the row class is to set up the bootstrap grid system. You don't have columns here. Not sure it's needed, but it doesn't hurt, either.